src/api/sizeme-api.js

```diff
@@ -373,6 +373,21 @@ function getRecommendedFit (fitResults, optimalFit) {
                 return [accSize, fit];
             }
         }, [null, 0]);
+    if (optFit == 1000) {
+        const optStretch = DEFAULT_OPTIMAL_STRETCH;
+        [bestMatch] = fitResults
+            .filter(([, res]) => res.totalFit >= 1000 && res.accuracy > 0)
+            .reduce(([accSize, fit], [size, res]) => {
+                let matchArr = Object.keys(res.matchMap).map((k) => res.matchMap[k]);
+                let maxStretch = Math.max.apply(Math, matchArr.map(function(o) { return o.componentStretch; }));
```

Comment:
`map`:ille on ehkä siistimpi antaa arrow-funktio: `.map(o => o.componentStretch)`. 
`apply`:n ensimmäinen argumentti voisi olla `null`, koska `max`-funktio ei tarvitse `this`:iä mihinkään